### ISSUE TITLE
Cant Install - Vim compiled with +python3 error

### ISSUE THREAD

**Issue by zen3gr** [puremourning/vimspector#464]
> DO NOT DELETE THIS TEMPLATE. IF YOU DELETE THIS TEMPLATE AND DO NOT COMPLETE IT, YOUR ISSUE WILL BE CLOSED.

### Describe the bug

> While using vim on Windows 10 (64bit) with latest version of Python (v3.9.7) I get this error while trying to install viminspector using vundle manager: "Requires vim compiled with +python3". When I run from cmd "vim --version" I get "+python\dyn +python3\dyn"; weird thing is I can run .py files from vim by typing :!python %.

### Minimal reproduciton

> Please answer the following questions

* Does your issue reproduce using `vim --clean -Nu /path/to/vimspector/support/minimal_vimrc` ? \[Yes/No] I can't reproduce it since problem is with installation.
* If you are using Neovim, does your issue reproduce using Vim? \[Yes/No] vim user.

> List of steps to reproduce:

> 1. Run `vim ---clean Nu /path/to/vimspector/support/minimal_vimrc`
> 2. Open _this project_...
> 3. Press _this sequence of keys_

> Use the following Vimspector config file:

```
paste .vimspector.json here
```

### Expected behaviour

> Provide A clear and concise description of what you expected to happen.

### Actual behaviour

> What actually happened, including output, log files etc.

> Please include:
> * Vimspector log (~/.vimspector.log)
> * Output from any or all UI diagnostic tabs (Server, etc.)

### Environemnt

***NOTE***: NeoVim is supported only on a best-effort basis. Please check the README
for limitations of neovim. Don't be offended if I ask you to reproduce issues in
Vim.

***NOTE***: Windows support is experimental and best-efrort only. If you find an
issue related to Windows or windows-isms, consider sending a PR or
discussing on Gitter rather than raising an issue.

* Version of Vimspector: (e.g. output of `git rev-parse HEAD` if cloned or the
  name of the tarball used to install otherwise)

* Output of `:VimspectorDebugInfo`

```
paste here
```

* Output of `vim --version` or `nvim --version`

```
VIM - Vi IMproved 8.2 (2019 Dec 12, compiled Sep 20 2021 22:03:42)
MS-Windows 64-bit console version
Included patches: 1-3451
Compiled by appveyor@APPVYR-WIN
Huge version without GUI.  Features included (+) or not (-):
+acl                +ex_extra           +multi_lang         -tag_any_white
+arabic             +extra_search       +mzscheme/dyn       +tcl/dyn
+autocmd            -farsi              -netbeans_intg      +termguicolors
+autochdir          +file_in_path       +num64              +terminal
+autoservername     +find_in_path       +packages           -termresponse
-balloon_eval       +float              +path_extra         +textobjects
+balloon_eval_term  +folding            +perl/dyn           +textprop
-browse             -footer             +persistent_undo    -tgetent
++builtin_terms     +gettext/dyn        +popupwin           +timers
+byte_offset        -hangul_input       -postscript         +title
+channel            +iconv/dyn          +printer            -toolbar
+cindent            +insert_expand      +profile            +user_commands
+clientserver       +ipv6               +python/dyn         +vartabs
+clipboard          +job                +python3/dyn        +vertsplit
+cmdline_compl      +jumplist           +quickfix           +virtualedit
+cmdline_hist       +keymap             +reltime            +visual
+cmdline_info       +lambda             +rightleft          +visualextra
+comments           +langmap            +ruby/dyn           +viminfo
+conceal            +libcall            +scrollbind         +vreplace
+cryptv             +linebreak          +signs              +vtp
+cscope             +lispindent         +smartindent        +wildignore
+cursorbind         +listcmds           -sodium             +wildmenu
+cursorshape        +localmap           +sound              +windows
+dialog_con         +lua/dyn            +spell              +writebackup
+diff               +menu               +startuptime        -xfontset
+digraphs           +mksession          +statusline         -xim
-dnd                +modify_fname       -sun_workshop       -xpm_w32
-ebcdic             +mouse              +syntax             -xterm_save
+emacs_tags         -mouseshape         +tag_binary
+eval               +multi_byte_ime/dyn -tag_old_static
   system vimrc file: "$VIM\vimrc"
     user vimrc file: "$HOME\_vimrc"
 2nd user vimrc file: "$HOME\vimfiles\vimrc"
 3rd user vimrc file: "$VIM\_vimrc"
      user exrc file: "$HOME\_exrc"
  2nd user exrc file: "$VIM\_exrc"
       defaults file: "$VIMRUNTIME\defaults.vim"
Compilation: cl -c /W3 /GF /nologo  -I. -Iproto -DHAVE_PATHDEF -DWIN32  -DFEAT_CSCOPE -DFEAT_TERMINAL -DFEAT_SOUND  -DFEAT_JOB_CHANNEL -DFEAT_IPV6        -DWINVER=0x0501 -D_WIN32_WINNT=0x0501 /source-charset:utf-8 /MP -DHAVE_STDINT_H /Ox /GL -DNDEBUG  /Zl /MT /D_CRT_SECURE_NO_DEPRECATE /D_CRT_NONSTDC_NO_DEPRECATE -DFEAT_MBYTE_IME -DDYNAMIC_IME -DDYNAMIC_ICONV -DDYNAMIC_GETTEXT -DFEAT_TCL -DDYNAMIC_TCL -DDYNAMIC_TCL_DLL=\"tcl86t.dll\" -DDYNAMIC_TCL_VER=\"8.6\" -DFEAT_LUA -DDYNAMIC_LUA -DDYNAMIC_LUA_DLL=\"lua53.dll\" -DFEAT_PYTHON -DDYNAMIC_PYTHON -DDYNAMIC_PYTHON_DLL=\"python27.dll\" -DFEAT_PYTHON3 -DDYNAMIC_PYTHON3 -DDYNAMIC_PYTHON3_DLL=\"python39.dll\" -DFEAT_MZSCHEME -I "C:\Program Files\Racket\include" -DMZ_PRECISE_GC -DDYNAMIC_MZSCHEME -DDYNAMIC_MZSCH_DLL=\"libracket3m_a36fs8.dll\" -DDYNAMIC_MZGC_DLL=\"libracket3m_a36fs8.dll\" -DFEAT_PERL -DPERL_IMPLICIT_CONTEXT -DPERL_IMPLICIT_SYS -DDYNAMIC_PERL -DDYNAMIC_PERL_DLL=\"perl528.dll\" -DFEAT_RUBY -DDYNAMIC_RUBY -DDYNAMIC_RUBY_DLL=\"x64-msvcrt-ruby240.dll\" -DRUBY_VERSION=24 -DFEAT_HUGE /Fd.\ObjCULYHTRZAMD64/ /Zi
Linking: link  /nologo /opt:ref /LTCG:STATUS oldnames.lib kernel32.lib advapi32.lib shell32.lib gdi32.lib  comdlg32.lib ole32.lib netapi32.lib uuid.lib /machine:AMD64   libcmt.lib  user32.lib  /nodefaultlib:lua53.lib  /STACK:8388608  /nodefaultlib:python27.lib /nodefaultlib:python39.lib   "C:\Tcl\lib\tclstub86.lib" winmm.lib WSock32.lib Ws2_32.lib /PDB:vim.pdb -debug
```

* Output of `which vim` or `which nvim`:

```
??windows user??
```

* Output of `:py3 print( __import__( 'sys' ).version )`:

```
Using minimalrc: 3.9.7 (tags/v3.9.7:1016ef3, Aug 30 2021, 20:19:38) [MSC v.1929 64 bit (AMD64)]

```

* Output of `:py3 import vim`:

```
Using minimalrc: (no output)
```

* Output of `:py3 import vimspector`:

```
Using minimalrc: Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'vimspector'

```

* For neovim: output of `:checkhealth`

```
---vim---
```

* Operating system: <linux or macOS> and version
  
```
Windows 10 64bit
```

### Declaration

> Please complete the following declaration. If this declaration is not completed, your issue may be closed without comment.

* I have read and understood [CONTRIBUTING.md](https://github.com/puremourning/vimspector/blob/master/CONTRIBUTING.md) \[Yes/No]

### Additional information

Thanks.

**Comment by puremourning** [comment]
I’m kind of confused 

Output of :py3 print( __import__( 'sys' ).version ):
Using minimalrc: 3.9.7 (tags/v3.9.7:1016ef3, Aug 30 2021, 20:19:38) [MSC v.1929 64 bit (AMD64)]

Output of :py3 import vim:
Using minimalrc: (no output)


This suggests that python is working. But I don’t understand why it would say ‘minimalrc’. If you are saying that with the minimal vimrc then py3 command works but vimspector doesn’t that’s a tricky one. 

But you said you ‘couldn’t reproduce with the minimal vimrc’ which isn’t true. Hence confused.

**Comment by zen3gr** [comment]
Excuse me for writing in such a confusing way. What I mean't was:
Output of :py3 print( import( 'sys' ).version ):
I got the following output using minimal_rc -> 3.9.7 (tags/v3.9.7:1016ef3, Aug 30 2021, 20:19:38) [MSC v.1929 64 bit (AMD64)]
---------
Anyway, I had this line on my main vimrc set pythonthreedll="C:\Users\myusername\AppData\Local\Programs\Python\Python39\python39.dll"
When I commented it, plugin was installed successfully. Thanks again!
